src/components/appointments/AppointmentForm.stories.mdx

```diff
@@ -0,0 +1,103 @@
+import {Meta, Canvas, Story, ArgsTable} from '@storybook/addon-docs';
+import {ConfigContext} from 'Context';
+import {MemoryRouter} from 'react-router-dom';
+
+import {Layout, LayoutRow} from 'components/Layout';
+
+import AppointmentForm from 'components/appointments/AppointmentForm';
+
+
+export const BASE_URL = 'http://localhost:8000/api/v2/';
+
+export const ConfigDecorator = (Story, {args}) => (
+  <ConfigContext.Provider value={{baseUrl: args.baseUrl || BASE_URL}}>
+    <Story />
+  </ConfigContext.Provider>
+);
+
+export const PaddedDiv = ({className, children}) => (
+  <div style={{padding: '1em 0'}} className={className}>{children}</div>
+);
+
+export const LayoutDecorator = (Story) => {
+  return (
+    <Layout component={PaddedDiv}>
+      <LayoutRow>
+        <Story />
+      </LayoutRow>
+    </Layout>
+  );
+};
+
+<Meta
+  title="Composites/Appointment form"
+  component={AppointmentForm}
+  parameters={{
+    mockData: [
```

Comment:
please rebase on `main` and set up the appropriate msw mocks, you can find some documentation here: https://open-formulieren.github.io/open-forms-sdk/?path=/story/developers-documentation--page#mocking-in-stories and some examples in https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/LanguageSelection/mocks.js and https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/LanguageSelection/LanguageSelection.stories.mdx?plain=1#L61